### PR TITLE
Resolve #2874

### LINKED ISSUE #2874: The list of the output does not coincide with the actual output for get-winuserlanguagelist
I think two pictures would suffice:
Expected output according to documentation:
image
Actual Output:
image

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fcfd899c-a63d-db98-8e39-aa226604432c
* Version Independent ID: 0603585f-7db5-8ccf-99ca-beef7cf671b4
* Content: [Get-WinUserLanguageList (International)](https://docs.microsoft.com/en-us/powershell/module/international/get-winuserlanguagelist?view=windowsserver2019-ps)
* Content Source: [docset/winserver2019-ps/international/Get-WinUserLanguageList.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/winserver2019-ps/international/Get-WinUserLanguageList.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @JasonGerend
* Microsoft Alias: **jgerend**

### PR DESCRIPTION
**Pull request by velkovb**
Correct output description
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2874

### CODE REVIEW

File: docset/winserver2012-ps/international/Get-WinUserLanguageList.md [hunk 1]
```diff
@@ -85,24 +85,33 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## OUTPUTS
 
 ### System.Collections.Generic.List<Microsoft.InternationalSettings.Commands.WinUserLanguage>
-A list of WinUserLanguage objects that contain one or more languages and associated properties from the current user account's language list.
-For information about the Generic.List object, see System.Collections.Generic.List<T\> Classhttp://go.microsoft.com/fwlink/?LinkID=243342.
+This cmdlet returns a list of **WinUserLanguage** objects that contain one or more languages and associated properties from the current user account's language list.
+For information about the **Generic.List** object, see [List(Of T) Class](https://go.microsoft.com/fwlink/?LinkID=243342).
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/international/Get-WinUserLanguageList.md:89]:
```suggestion
For information about the **Generic.List** object, see [List<T> Class](/dotnet/api/system.collections.generic.list-1).
```

Comment (JohanFreelancer9) [inline docset/winserver2012-ps/international/Get-WinUserLanguageList.md:89]:
```suggestion
For information about the **Generic.List** object, see [List<T> Class](https://go.microsoft.com/fwlink/?LinkID=243342).
```

File: docset/winserver2012r2-ps/international/Get-WinUserLanguageList.md [hunk 2]
```diff
@@ -87,24 +87,33 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## OUTPUTS
 
 ### System.Collections.Generic.List<Microsoft.InternationalSettings.Commands.WinUserLanguage>
-A list of WinUserLanguage objects that contain one or more languages and associated properties from the current user account's language list.
-For information about the Generic.List object, see List(Of T) Classhttp://go.microsoft.com/fwlink/?LinkID=243342.
+This cmdlet returns a list of **WinUserLanguage** objects that contain one or more languages and associated properties from the current user account's language list.
+For information about the **Generic.List** object, see [List(Of T) Class](https://go.microsoft.com/fwlink/?LinkID=243342).
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/international/Get-WinUserLanguageList.md:91]:
```suggestion
For information about the **Generic.List** object, see [List<T> Class](https://go.microsoft.com/fwlink/?LinkID=243342).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @velkovb, kindly commit the two suggested changes if you agree with them. Many thanks.

File: docset/winserver2012-ps/international/Get-WinUserLanguageList.md [hunk 3]
```diff
@@ -85,24 +85,33 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## OUTPUTS
 
 ### System.Collections.Generic.List<Microsoft.InternationalSettings.Commands.WinUserLanguage>
-A list of WinUserLanguage objects that contain one or more languages and associated properties from the current user account's language list.
-For information about the Generic.List object, see System.Collections.Generic.List<T\> Classhttp://go.microsoft.com/fwlink/?LinkID=243342.
+This cmdlet returns a list of **WinUserLanguage** objects that contain one or more languages and associated properties from the current user account's language list.
+For information about the **Generic.List** object, see [List<T> Class](https://go.microsoft.com/fwlink/?LinkID=243342).
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/international/Get-WinUserLanguageList.md:89]:
```suggestion
For information about the **Generic.List** object, see [List(Of T) Class](https://go.microsoft.com/fwlink/?LinkID=243342).
```

File: docset/winserver2012r2-ps/international/Get-WinUserLanguageList.md [hunk 4]
```diff
@@ -87,24 +87,33 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## OUTPUTS
 
 ### System.Collections.Generic.List<Microsoft.InternationalSettings.Commands.WinUserLanguage>
-A list of WinUserLanguage objects that contain one or more languages and associated properties from the current user account's language list.
-For information about the Generic.List object, see List(Of T) Classhttp://go.microsoft.com/fwlink/?LinkID=243342.
+This cmdlet returns a list of **WinUserLanguage** objects that contain one or more languages and associated properties from the current user account's language list.
+For information about the **Generic.List** object, see [List<T> Class](https://go.microsoft.com/fwlink/?LinkID=243342).
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/international/Get-WinUserLanguageList.md:91]:
```suggestion
For information about the **Generic.List** object, see [List(Of T) Class](https://go.microsoft.com/fwlink/?LinkID=243342).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @velkovb, my bad. Please change back again. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Pull request merged**